File: csharp/raindrops/Raindrops.cs
using System;

public static class Raindrops
{
    public static string Convert(int number)
    {
        string result = number % 3 == 0 ? "Pling" : "";
        result += number % 5 == 0 ? "Plang" : "";
        result += number % 7 == 0 ? "Plong" : "";
        return result.Length > 0 ? result : number.ToString();
    }
}

File: javascript/raindrops/raindrops.js
export const convert = (input) => {
  return [...pairs.keys()]
          .filter(key => input % key === 0)
          .map(key => pairs.get(key))
          .join('')
          || input.toString();
};

const pairs = new Map([
  [3, 'Pling'],
  [5, 'Plang'],
  [7, 'Plong']
]);